### PR TITLE
Textarea

### PR DESCRIPTION
**Pull request by TudorTacal**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
```
Comment (TudorTacal) [inline src/App.js]: Do you need this logo anymore?

File: src/App.js [hunk 2]
```diff
@@ -10,8 +12,8 @@ class App extends Component {
           <img src={logo} className="App-logo" alt="logo" />
```
Comment (TudorTacal) [inline src/App.js]: you could extract this Header into a separate component

File: src/App.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
+import Text from './Text'
 import './App.css';
+import Previewer from './Previewer';
 
 class App extends Component {
```
Comment (TudorTacal) [inline src/App.js]: if you don't have state you can transform this components into a SFC - stateless function component

File: src/Previewer.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    border-radius: 20px;
+    padding: 0.25em;
+    margin: 2em;
+    margin-left: 25px;
+    margin-right: 25px;
+    align: center;
+    text-align: center;
+    font-family: arial;
+`;
+
```
Comment (TudorTacal) [inline src/Previewer.js:25]: good work on styling

File: src/Previewer.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+const StyledTitle = styled.h3`
+    text-align: center;
+`;
+
+const StyledPreviewer = styled.textarea`
+    width: 80%;
+    height: 70%;
+`;
+
+class Previewer extends React.Component {
```
Comment (TudorTacal) [inline src/Previewer.js]: again, you can transform into an SFC if you don't have state

Comment (TudorTacal) [reply src/Previewer.js]: something like this 
const Previewer = () => {
 return (
            <StyledContainer>
                <StyledTitle>Previewer</StyledTitle>
                <StyledPreviewer />
            </StyledContainer>
        )
}
But please google it, read about it and tell me why we should use a SFC over the class syntax.

**Comment by TudorTacal** [comment]
More attention to typos in the commit messages.
Use present tense.
When you refer to a component you can use capital letters, ex: style the Previewer component.

File: src/Previewer.js (hunk 4 above)
Comment (Lorenatacal) [reply src/Previewer.js:25]: Thank you

**Comment by Lorenatacal** [comment]
I have made all the changes that were required.What should I do next in order for you to see the changes that I have made?

**Pull request closed without merging**